src/nuclio/projects/project/functions/create-function/function-from-template/function-from-template.tpl.html

```diff
@@ -23,11 +23,38 @@
     </form>
     <div class="templates-wrapper">
         <span class="title">Choose a template</span>
+        <div class="templates-controls">
+            <div class="templates-search-input">
+                <div class="igz-icon-search search-icon"></div>
+                <input class="input-field field" tabindex="0"
+                       data-ng-model="$ctrl.searchQuery"
+                       data-ng-change="$ctrl.onChangeSearchQuery()"
+                       placeholder="Search by text, tags and keywords..."
+                       data-igz-input-blur-on-enter>
+            </div>
+            <div class="templates-runtime-drop-down">
+                <span class="input-label">Runtime</span>
+                <igz-default-dropdown data-values-array="$ctrl.runtimeFilters"
+                                      data-selected-item="$ctrl.selectedRuntimeFilter"
+                                      data-item-select-callback="$ctrl.onRuntimeFilterChange(item, isItemChanged)">
+                </igz-default-dropdown>
+            </div>
+            <div class="templates-pagination">
+                <igz-pagination data-page-data="$ctrl.page"
+                                data-is-per-page-visible="true"
+                                data-pagination-callback="$ctrl.paginationCallback(page, size)">
+                </igz-pagination>
```

Comment:
What is this component? The module `iguazio.dashboard-controls` does not know it.
If you use Nuclio inside Iguazio then it will work, since Iguazio has that component.
But if you just run Nuclio stand-alone the paginator will not be displayed.

If this component is now shared among Iguazio and Nuclio – please move it to dashboard-controls repo.